templates/watcher/config/00-default.conf

```diff
@@ -56,5 +58,13 @@ datasources = ceilometer
 
 {{ if (index . "MemcachedServers") }}
 [cache]
-memcached_servers = {{ .MemcachedServers }}
-{{ end }}
+{{if .MemcachedTLS}}
+backend = dogpile.cache.pymemcache
```

Comment:
how do we now which value to pick here? I'm not really sure which one we should pick based on the conf docs https://docs.openstack.org/watcher/latest/configuration/watcher.html#cache.backend

Reply:
yep, that's great, thanks for the explanation!